Scavenge-UW/Scavenge
# Implement logout in frontend

Implements logout in the frontend.

**How to Test**
1. Login with a valid credential
2. Go to Manage Pantry (or any other menu)
3. Click on logout
4. User is redirected back to home and the menu should show Login and Signup

**Change Summary**
Implements logout in the frontend.
